app/processing/components/get-schedule-dates.js

```diff
@@ -7,27 +9,29 @@ const getScheduleDates = (previousPaymentSchedule, newPaymentSchedule, deltaValu
 
   const paidSegments = previousPaymentSchedule.filter(x => !x.outstanding)
   newPaymentSchedule.splice(0, paidSegments.length)
-  if (deltaValue < 0) {
+  const nonAdjustmentValue = 0
+
+  if (deltaValue < nonAdjustmentValue) {
     // we need to avoid a balloon reduction, so we spread the reduction for any paid segments across remaining segments only
```

Comment:
should this comment be left in if we are typically removing comments from other files?

Reply:
The main purpose of this ticket is to make ffc-doc-statement-constructor be a replica of ffc-pay-statement-constructor aside for configuration values that are specific to the repository/service namespace. Any other change outside the original scope should be treated as a new ticket.

Reply:
Ok no problem on both comments, i'll create a new ticket for post migration cleanup and add these comments to that before approving the as-is codebase